tests/db/conftest.py

```diff
@@ -108,13 +108,13 @@ def inputs(user_vk, amount=1, b=None):
         pass
 
     # 2. create block with transactions for `USER` to spend
-    transactions = []
-    for i in range(amount):
-        tx = b.create_transaction(b.me, user_vk, None, 'CREATE')
-        tx_signed = b.sign_transaction(tx, b.me_private)
-        transactions.append(tx_signed)
-        b.write_transaction(tx_signed)
-
-    block = b.create_block(transactions)
-    b.write_block(block, durability='hard')
-    return block
+    for block in range(blocks):
+        transactions = []
+        for i in range(amount):
+            tx = b.create_transaction(b.me, user_vk, None, 'CREATE')
+            tx_signed = b.sign_transaction(tx, b.me_private)
+            transactions.append(tx_signed)
+            b.write_transaction(tx_signed)
```

Comment:
This may create problems. We are writing transactions to the backlog and then we are already creating blocks with the same transactions.